### PR TITLE
Fix/sd webui sampler index

### LINKED ISSUE #4717: The default name of stable_diffusion sampler_index has been changed
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.6.9

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

  DRAW_TEXT_OPTIONS.sampler_index in the stable_diffuse.py file, where the default name is "DPM++ SDE Karras", However, the sd webui has changed its default name to "DPM++ SDE" in the last 2024.3.24 commit record, so the api access has been unsuccessful.
  It is recommended to remove the sampler_index key and use the default sd value when passing empty values

### ✔️ Expected Behavior

The sd tool should be called successfully

### ❌ Actual Behavior

sd tool call should fail

### PR DESCRIPTION
**Pull request by lingfengchencn**
# Description

Following the suggestion from #4717, temporarily commented out sampler_index to be compatible with both the old and new versions of the SD WebUI API.

Fixes #4717 

## Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
test  at sd-webui version. 1.6(which is work fine before ) and 1.9.3 (the newest one)

### CODE REVIEW

File: api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ parameter_rules:
     use_template: max_tokens
     default: 1024
     min: 2
-    max: 1024
+    max: 2048
```
Comment (Yeuoly) [inline api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml:25]: hi, looks like this is a irrelevant change

File: api/core/tools/provider/builtin/stablediffusion/tools/stable_diffusion.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "subseed": -1,
     "subseed_strength": 0,
     "seed_resize_from_h": -1,
-    'sampler_index': 'DPM++ SDE Karras',
+    # 'sampler_index': 'DPM++ SDE Karras',
```
Comment (Yeuoly) [inline api/core/tools/provider/builtin/stablediffusion/tools/stable_diffusion.py:23]: what about introduce a new parameter named `samlper_index` as different models depend on different sampler.

**Review by Yeuoly — CHANGES_REQUESTED** [review]

File: api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml (hunk 1 above)
Comment (lingfengchencn) [reply api/core/model_runtime/model_providers/wenxin/llm/ernie-lite-8k-0308.yaml:25]: oh no, I will change it.

File: api/core/tools/provider/builtin/stablediffusion/tools/stable_diffusion.py (hunk 2 above)
Comment (lingfengchencn) [reply api/core/tools/provider/builtin/stablediffusion/tools/stable_diffusion.py:23]: ok,I will turn it into a parameter and look forward to the frontend team making it a configurable value.

**Pull request closed without merging**